src/main/java/com/napier/sem/App.java

```diff
@@ -23,19 +23,19 @@ public static void main(String[] args) {
         // Connect to database
         if (args.length < 1)
         {
-            connection = connect("localhost:3306");
+            connection = connect("localhost:33061", "com.mysql.cj.jdbc.Driver");
```

Comment:
`"com.mysql.cj.jdbc.Driver"` should probably be a constant, happy to change this if we want